### PR TITLE
Merak Topology

### PR DESCRIPTION
**Pull request by zzengcs**
Ready for review

### CODE REVIEW

File: api/proto/v1/common.proto [hunk 1]
```diff
@@ -60,7 +60,8 @@ message InternalComputeInfo {
   string ip = 4;
   string mac = 5;
   string veth = 6;
-  Status status = 7;
+  string container_ip = 7;
+  Status status = 8;
```
Comment (cj-chung) [inline api/proto/v1/common.proto:64]: @zzengcs I will update this file, please remove this file from your commit.

File: docs/components/merak-topo.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
+<!-- /*
+MIT License
+Copyright(c) 2022 Futurewei Cloud
+    Permission is hereby granted,
+    free of charge, to any person obtaining a copy of this software and associated documentation files(the "Software"), to deal in the Software without restriction,
+    including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and / or sell copies of the Software, and to permit persons
+    to whom the Software is furnished to do so, subject to the following conditions:
+    The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
```
Comment (cj-chung) [inline docs/components/merak-topo.md:8]: @zzengcs We don't need license header in the doc or md files

**Review by cj-chung — CHANGES_REQUESTED** [review]

File: api/proto/v1/common.proto (hunk 1 above)
Comment (zzengcs) [reply api/proto/v1/common.proto]: Fixed

File: docs/components/merak-topo.md (hunk 2 above)
Comment (zzengcs) [reply docs/components/merak-topo.md]: Fixed, tks

File: services/merak-topo/handler/topo_deploy.go [hunk 3]
```diff
@@ -0,0 +1,417 @@
...
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+
+	"k8s.io/client-go/tools/remotecommand"
+)
+
+var (
+	ACA_IMAGE = "phudtran/merak-agent:dev"
+	OVS_IMAGE = "yanmo96/ovs_only:latest"
+	GW_IMAGE  = "yanmo96/aca_build_standard:v2"
```
Comment (cj-chung) [inline services/merak-topo/handler/topo_deploy.go:43]: @zzengcs Please make a comments to indicate that we need to make these Image uri as input parameters or get these from `service_config` protobuf message.

**Review by cj-chung — APPROVED** [review]
LGTM

**Comment by cj-chung** [comment]
@zzengcs You have two conflict, please remove the modification for these two files.

**Pull request closed without merging**